tests/node_configuration.rs

```diff
@@ -93,19 +93,6 @@ mod test_rand {
         config.base_token.unit = "suSHI".to_string();
         config.base_token.subunit = "rice".to_string();
 
-        // rejected change (not latest)
-        node_configuration
-            .update_latest_node_configuration(1.into(), config.clone())
-            .await
-            .unwrap();
-        assert_eq!(node_configuration.count().await.unwrap(), 1);
-        assert_eq!(&latest_config.config.base_token.name, "Shimmer");
-        assert_eq!(&latest_config.config.base_token.ticker_symbol, "SMR");
-        assert_eq!(&latest_config.config.base_token.unit, "SMR");
-        assert_eq!(&latest_config.config.base_token.subunit, "glow");
-        assert_eq!(latest_config.config.base_token.decimals, 6);
-        assert!(latest_config.config.base_token.use_metric_prefix);
-
```

Comment:
I don't think we should delete this test case. We should rather "sandwich" a meaningless configuration _t1_ (where nothing has changed) between two configurations at _t0_ and _t2_. Ideally, _t0_ would be inserted after _t2_ to mimic our logic for catching up.